subsys/bluetooth/audio/endpoint.h

```diff
@@ -73,6 +73,7 @@ struct bt_audio_unicast_group {
 struct bt_audio_broadcast_source {
 	uint8_t stream_count;
 	uint8_t subgroup_count;
+	bool adv_started;
```

Comment:
Where is this one used?
In the functions above, a local variable seems to be used.

Reply:
Ah, good catch. I started implementing with putting it here, but then figured that it better to extract the state from the endpoint states.